ow_sim_tests/scripts/test_pan_tilt_move_joints.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+import roslib
+import unittest
+import owl_msgs.msg
+import ow_lander.msg
+from action_testing import *
+
+PKG = 'ow_sim_tests'
+TEST_NAME = 'pan_tilt_move_joints'
+roslib.load_manifest(PKG)
+
+class TestPanTiltMoveJoints(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(cls):
+    rospy.init_node("pan_tilt_move_joints_test")
+
+    set_ignore_action_checks(True)
+    # proceed with test only when ros clock has been initialized
+    while rospy.get_time() == 0:
+      rospy.sleep(30.0)
```

Comment:
Same change here please
```suggestion
    while rospy.get_time() == 0:
      rospy.sleep(0.1)
    # NOTE: work around for pan/tilt actions failing when called too early (see OW-1181)
    rospy.sleep(30.0)
```